onnx-ops/import_onnx_tests.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import onnx
+from multiprocessing import Pool
+from pathlib import Path
+from onnx import version_converter
+import shutil
+import subprocess
+import sys
+from import_onnx_tests_utils import *
+
+
+ONNX_PACKAGE_DIR = Path(onnx.__file__).parent
+ONNX_NODE_TESTS_ROOT = ONNX_PACKAGE_DIR / "backend/test/data/node"
```

Comment:
Thanks for noticing that a git submodule isn't needed here, @zjgarvey !

Reply:
BTW, this `__file__` solution is a bit cleaner and more robust than https://github.com/nod-ai/SHARK-TestSuite/blob/3b22038d687a13e3c46a8575d3e465133dfcbb7e/alt_e2eshark/onnx_tests/operators/generate_node.py#L29-L32

```python
from site import getsitepackages

base_dir = getsitepackages()[0]
# you can also get the node tests from the git submodule instead by uncommenting:
# base_dir = str(Path(__file__).parents[3]) + "/third_party/onnx"
onnx_node_tests_dir = base_dir + "/onnx/backend/test/data/node/"
```

In particular, there may be multiple site packages directories, and that assumed that onnx was in the first.

Reply:
Yeah, that is much cleaner. I'll make that change over in shark test suite, too.